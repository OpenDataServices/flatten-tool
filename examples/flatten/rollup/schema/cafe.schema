{
    "$schema": "http://json-schema.org/draft-04/schema#",
    "definitions": {
        "OwnerObject": {
            "type": "object",
            "properties": {
                "firstname": {
                    "type": "string",
                    "title": "First name"
                },
                "lastname": {
                    "type": "string",
                    "title": "Last name"
                },
                "email": {
                    "type": "string",
                    "title": "Email"
                }
            }
        },
        "DishObject": {
            "type": "object",
            "properties": {
                "id": {
                    "type": "string",
                    "title": "Identifier"
                },
                "title": {
                    "type": "string",
                    "title": "Name"
                },
                "allergens": {
                    "type": "object",
                    "title": "Allergens",
                    "properties": {
                        "freefrom": {
                            "type": "array",
                            "title": "Free from",
                            "items": {
                                "type": "string"
                            }
                        },
                        "contains": {
                            "type": "array",
                            "title": "Contains",
                            "items": {
                                "type": "string"
                            }
                        },
                        "label": {
                            "type": "array",
                            "title": "Label",
                            "items": {
                                "type": "string"
                            }
                        }
                    }
                }
            }
        }
    },
    "type": "object",
    "properties": {
        "id": {
            "type": "string",
            "title": "Identifier"
        },
        "type": {
            "type": "string",
            "title": "Type"
        },
        "title": {
            "type": "string",
            "title": "Name"
        },
        "address": {
            "type": "string",
            "title": "Address"
        },
        "owners": {
            "items": {
                "$ref": "#/definitions/OwnerObject"
            },
            "type": "array",
            "title": "Owner"
        },
        "dishes": {
            "items": {
                "$ref": "#/definitions/DishObject"
            },
            "type": "array",
            "title": "Dish"
        },
        "coffee": {
            "type": "string",
            "title": "Coffee quality"
        },
        "coke": {
            "type": "string",
            "title": "Type of coke"
        }
    }
}
