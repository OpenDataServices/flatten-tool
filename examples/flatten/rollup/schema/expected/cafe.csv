Identifier,Type,Name,Address,Owner:First name,Owner:Last name,Owner:Email,Coffee quality,Type of coke
1,cafe,Cafe Open,,Clark,Kent,contact@cafeopen.example,passable,
2,pub,Pints R Us,,Jim,Kirk,pintsrus@protonmail.com,,pepsi
